src/jti_background.js

```diff
@@ -77,6 +79,35 @@ function getData (callback) {
     });
 }
 
+// Get toggle status based on 'toggleType'
+function getToggleStatus (toggleType, callback) {
+    chrome.storage.sync.get(StorageToggleID, function (toggles) {
+        if (jQuery.isEmptyObject(toggles)) { // If user does not have any toggle settings in storage
+            callback(false, 'No data is currently loaded');
+        } else {
+            if (toggles[StorageToggleID][toggleType]) {
+                callback(true, '', toggles[StorageToggleID][toggleType]);
+            } else {
+                callback(false, 'No data is currently loaded');
+            }
+        }
+    });
+}
+
+// Set toggle status based on 'toggleType'
+function setToggleStatus (toggleType, callback) {
```

Comment:
Set toggle should probably accept an input as the value to set the toggle to. That way we could set the toggle to an arbitrary value. Not simply "true".